### ISSUE TITLE
Use different strings for `Add word` and `Word`

### ISSUE THREAD

**Issue by e-me** [otacke/h5p-crossword#11]
Due to the fact that the Greek language (and other languages too) has cases, the same field cannot be used in both cases.
As one of them has to be nominative and the latter one genitive.

`H5P.Crossword-0.4\language\el.json -- line 14, line 16`

**Comment by otacke** [comment]
Sorry, I am not sure what you are hinting at. You are referencing https://github.com/otacke/h5p-crossword/blob/master/language/.en.json#L14 and https://github.com/otacke/h5p-crossword/blob/master/language/.en.json#L16 in the Greek version? Why don't you supply a fixed translation then?

If it's a general issue with how these translations are displayed in the editor, then you'll have to address the H5P core team, as I cannot change how the editor interprets the semantics file and applies the translations.

**Comment by e-me** [comment]
Yeah, sorry for the poor explanation, let me try again,
It seems that the same string is used in both cases `Word` and `Add Word`. If we had separate ones would resolve the issue.

![image](https://user-images.githubusercontent.com/38039666/133794709-ef0714c1-3ba4-46b4-a8e6-d48f033ac777.png)

The same issue exists in `sort the paragraphs` as well by the way.

**Comment by otacke** [comment]
That's a problem with the H5P editor, not with Crossword. In particular, it's the [vertical tabs widget](https://github.com/h5p/h5p-editor-vertical-tabs) that uses the "entity" for the tabs (https://github.com/h5p/h5p-editor-vertical-tabs/blob/master/vertical-tabs.js#L283) and for the button (https://github.com/h5p/h5p-editor-vertical-tabs/blob/master/vertical-tabs.js#L32). That's nothing that I'll fix exclusively for Crossword.

**Comment by e-me** [comment]
Ok, I see, thank for the explanation.
Do you think it's possible to be implemented by the h5p core team? If so how should I describe properly the issue?

**Comment by otacke** [comment]
Sure. You'd simply need to describe what I outlined in my previous answer, that the [vertical tabs widget](https://github.com/h5p/h5p-editor-vertical-tabs) uses the `entity` property of a `list` for both - the label of a tab (https://github.com/h5p/h5p-editor-vertical-tabs/blob/master/vertical-tabs.js#L283) and for the "add entity" button text (https://github.com/h5p/h5p-editor-vertical-tabs/blob/master/vertical-tabs.js#L32) and that this may cause issues with Grammar in different languages. A separate field for a `list` might be useful, could also be a separate field for the vertical tabs widget parameters if it won't get purged by the h5p cli tool, etc.

**Comment by e-me** [comment]
Thanks for the help, closing issue.
